fluentd/configs.d/filter-pre-mux-client.conf

```diff
@@ -5,6 +5,8 @@
   shared_key "#{File.open('/etc/fluent/muxkeys/shared_key') do |f| f.readline end.rstrip}"
   secure yes
   ca_cert_path /etc/fluent/muxkeys/ca
+  buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT']}"
```

Comment:
Wouldn't 1 MB chunks be sufficient, with a `flush_interval` of 1 s to try to keep the queue short?

Additionally, if we need a bigger queue, then we'll want to use the `file` buffer instead of `memory` buffer.